adafruit_rtttl.py

```diff
@@ -40,6 +40,10 @@
     import audioio
     from adafruit_waveform import sine
     AUDIOIO_AVAILABLE = True
+    try:
+        import audiocore
+    except ImportError:
+        audiocore = audioio
```

Comment:
i think you want to do `import audioio as audiocore` intead of two imports?

Reply:
I'm not sure what you mean.
Two imports are needed with 5.0.0 both audioio and audiocore.

Reply:
ooh ok!